src/live/index.js

```diff
@@ -49,6 +50,22 @@ const onConnect = (io, socket) => {
       message: 'User paused the album',
     });
   });
+
+  socket.on('listening-to-album-play', () => {
+    // albumId should be added as a parameter   socket.on('listening-to-album-play', (albumId) => {
+    //   albumController.startAlbumPlayback(albumId);
+    console.log('Album is now playing');
+  });
+
+  socket.on('listening-to-album-pause', () => {
+    io.emit('album-paused', 'Album is now paused');
+    console.log('Album is now paused');
+  });
```

Comment:
After your latest merge/commit I'm seeing two `listening-to-album-pause` and `listening-to-album-play` handlers here. Please remove one (probably these two highlighted can be removed for now)

Reply:
Removed duplicated handlers, commit 4a3c59816c9eb9ca69a9cfb1002af8b074554931